### PR TITLE
Add optional fields to company search-result candidate

### PR DESCRIPTION
**Pull request by thomassa**

### CODE REVIEW

**Comment by thomassa** [comment]
I've not tested this: I was hoping we would have something automatic that would run on the pullreq branch and would let us view the output.

File: source/includes/_08-additional-features.md.erb [hunk 1]
```diff
@@ -369,8 +377,69 @@ app.post("/search", (req, res) => {
...
+      </td>
+      <td>No</td>
+      <td>
+        Exact date in YYYY-MM-DD format
+      </td>
+    </tr>
+    <tr>
+      <td><code>structure_type</code></td>
+      <td>
+        Object containing structured fields
```
Comment (thomassa) [inline source/includes/_08-additional-features.md.erb:416]: In the light of @louisdewar's impending changes https://github.com/PassFort/PassFort/pull/9061 defining StructuredCompanyType in the OpenAPI-format description of the PassFort API, we should refer to that type here.

**Review by thomassa — COMMENTED** [review]
Should refer to the new StructuredCompanyType being added in the PassFort repo.

File: source/includes/_08-additional-features.md.erb [hunk 2]
```diff
@@ -369,8 +377,69 @@ app.post("/search", (req, res) => {
...
+      <td>No</td>
+      <td>Global Legal Entity Identifier</td>
+    </tr>
+    <tr>
+      <td><code>tax_ids</code></td>
+      <td>List of TaxId objects</td>
+      <td>No</td>
+      <td>
+        Tax identifiers; each TaxId object is structured as in <a
+        href="#data-structure">the PassFort API</a>
```
Comment (louisdewar) [inline source/includes/_08-additional-features.md.erb:439]: missing full-stop (it seems like the description fields have full-stops but not the type fields

**Review by louisdewar — APPROVED** [review]
The full-stop comment is pretty minor, and I doubt we've been consistent before.
All the new fields are here with appropriate descriptions.
Also it looks like you found and fixed some pre-existing HTML issues.

Even though the code looks fine I think we should still try to test it locally, have you tried the instructions in the README?

File: source/includes/_08-additional-features.md.erb [hunk 3]
```diff
@@ -369,8 +377,69 @@ app.post("/search", (req, res) => {
...
+      <td>
+        Object containing structured fields
+      </td>
+      <td>No</td>
+      <td>
+        Object structured the same as the `structured_company_type` field of
+        the `CompanyMetadata` object in <a href="#data-structure">the PassFort
+        API</a>: "Structured fields indicating the kind of liability its
+        members are exposed to, the ownership structure, and whether it is
+        publically traded."
```
Comment (JennyShuZhan) [inline source/includes/_08-additional-features.md.erb:424]: Is it `publicly`?

File: source/includes/_08-additional-features.md.erb [hunk 4]
```diff
@@ -369,8 +377,69 @@ app.post("/search", (req, res) => {
...
+      <td>string</td>
+      <td>No</td>
+      <td>Global Legal Entity Identifier</td>
+    </tr>
+    <tr>
+      <td><code>tax_ids</code></td>
+      <td>List of TaxId objects</td>
+      <td>No</td>
+      <td>
+        Tax identifiers; each TaxId object is structured as in <a
```
Comment (JennyShuZhan) [inline source/includes/_08-additional-features.md.erb:438]: The ; is a bit inconsistent with the other ones which are with ,

File: source/includes/_08-additional-features.md.erb (hunk 3 above)
Comment (thomassa) [reply source/includes/_08-additional-features.md.erb:424]: Yes; good catch. I had just copied this text from the main PassFort API, so we should fix it there too.

File: source/includes/_08-additional-features.md.erb (hunk 4 above)
Comment (thomassa) [reply source/includes/_08-additional-features.md.erb:438]: "," would be wrong here. Which others do you mean?

File: source/includes/_08-additional-features.md.erb (hunk 1 above)
Comment (thomassa) [reply source/includes/_08-additional-features.md.erb]: Done.

File: source/includes/_08-additional-features.md.erb (hunk 3 above)
Comment (thomassa) [reply source/includes/_08-additional-features.md.erb]: Fixed.

File: source/includes/_08-additional-features.md.erb (hunk 4 above)
Comment (thomassa) [reply source/includes/_08-additional-features.md.erb]: Anyway, I've changed the wording so as to avoid the issue.

**Review by JennyShuZhan — APPROVED** [review]

**Comment by thomassa** [comment]
(I've re-done the commits with gpg signatures, since this repo demands them.)

**Comment by thomassa** [comment]
I've looked at the github actions configured for this repo, and realised that as soon as we merge it to master, the "publish_docs" workflow will be run and will use https://github.com/PassFort/middleman-gh-pages-action to build the docs and publish them to the github pages.

I don't think we don't want to publish the update yet, do we?
Not until we've deployed the versions of the bridge and monolith that can cope with the extra fields.
So for now I've changed this pullreq to draft status so that it doesn't get merged prematurely.
I don't anticipate making any further changes to it though.

(Also before merging I'd like to test locally it by doing essentially the same as that "publish_docs" workflow, using the docker image etc.)

**Pull request merged**